### ISSUE TITLE
Dockerfile/docker build broken

### ISSUE THREAD

**Issue by lanmalkieri** [coinbase/terraform-landscape#96]
Currently docker build does not work with a latest pull from master. Docker build results in the following error. 

```
aa|[prod][terraform-landscape:master][~/git/landscape] > docker build . -t landscape
Sending build context to Docker daemon  1.649MB
Step 1/4 : FROM alpine:3.8
 ---> dac705114996
Step 2/4 : RUN apk --no-cache add   ruby-bundler=1.16.2-r1   ruby-json=2.5.2-r0   diffutils=3.6-r1 # this is required for diffy to work on alpine
 ---> Running in aad9e591549b
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  ruby-json-2.5.5-r0:
    breaks: world[ruby-json=2.5.2-r0]
The command '/bin/sh -c apk --no-cache add   ruby-bundler=1.16.2-r1   ruby-json=2.5.2-r0   diffutils=3.6-r1 # this is required for diffy to work on alpine' returned a non-zero code: 1
```

**Comment by sds** [comment]
Would welcome a pull request fixing!

**Comment by sds** [comment]
Fixed in #97.

**Comment by lanmalkieri** [comment]
Hey thanks @sds

Sorry, just seeing your response here or would have submitted a pr.
